### DISCUSSION TITLE
What is the equivalent FastApi way to do request.json() as in Flask

### DISCUSSION THREAD

**Discussion by steinskeeper** [fastapi/fastapi#6824] [Questions] [answered]
In Flask,requests from client can be handled like this
For JSON Data

`payload = request.get_json()`

For token arguments

`token = request.headers.get('Authorization')`

For arguments 

`id = request.args.get('url', None)`

What is the FastApi way of doing the same? (Without Pydantic)

**Comment by ycd** [comment, marked as answer]
We have a documentation about it. See: https://fastapi.tiangolo.com/advanced/using-request-directly/

```python
from fastapi import Request, FastAPI

@app.post("/dummy")
async def get_body(request: Request):
    return await request.body()
```

**Comment by tiangolo** [comment]
Thanks for the help here @ycd ! 👏  🙇 

Thanks for reporting back and closing the issue @steinskeeper  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
